### ISSUE TITLE
Building sqlite/sqlx features implicitly enables runtimes

### ISSUE THREAD

**Issue by yaleman** [maxcountryman/axum-login#5]
Is there a way to disable this? I'm trying to use axum-login in an existing tokio/axum project and enabling either of the sqlite/sqlx features produces an error:

```
06:17:48 ❯ cargo build
   Compiling sqlx-rt v0.6.2
error: only one of ['runtime-actix-native-tls', 'runtime-async-std-native-tls', 'runtime-tokio-native-tls', 'runtime-actix-rustls', 'runtime-async-std-rustls', 'runtime-tokio-rustls'] can be enabled
  --> /Users/yaleman/.cargo/registry/src/github.com-1ecc6299db9ec823/sqlx-rt-0.6.2/src/lib.rs:23:1
   |
23 | / compile_error!(
24 | |     "only one of ['runtime-actix-native-tls', 'runtime-async-std-native-tls', \
25 | |      'runtime-tokio-native-tls', 'runtime-actix-rustls', 'runtime-async-std-rustls', \
26 | |      'runtime-tokio-rustls'] can be enabled"
27 | | );
   | |_^

error: could not compile `sqlx-rt` due to previous error
warning: build failed, waiting for other jobs to finish...
error: could not compile `sqlx-rt` due to previous error
```

**Comment by maxcountryman** [comment]
That sounds like a pain, sorry you're running into this.

I'm happy to make that optional assuming that's feasible with the right set of feature flags.

**Comment by yaleman** [comment]
I figured out later that changing sqlx features in my project from `runtime-native-rustls` to `runtime-tokio-rustls` sorted it for now, which is probably smarter, but it's likely an issue either way for adoption 😃

**Comment by maxcountryman** [comment]
@yaleman if we remove the flag entirely, does that address the issue?

```diff
diff --git a/Cargo.toml b/Cargo.toml
index 25f3b09..7c233b5 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -25,7 +25,6 @@ mssql = ["sqlx/mssql"]
 mysql = ["sqlx/mysql"]
 postgres = ["sqlx/postgres"]
 sqlite = ["sqlx/sqlite"]
-sqlx = ["sqlx/runtime-tokio-rustls"]
 
 [dependencies]
 async-trait = "0.1.57"
```

Applications will need to specify the runtime (which must be tokio) and TLS preference.

**Comment by yaleman** [comment]
I'm no longer using this crate in the project so I'm not able to test, sorry!

**Comment by maxcountryman** [comment]
@yaleman got it. Any particular feedback re moving away from this crate?

**Comment by maxcountryman** [comment]
Makes sense!

**Comment by maxcountryman** [comment]
There's some work to address this in #44 but I think we need to resolve #73 before we can figure out the best approach here.

**Comment by maxcountryman** [comment]
This should be addressed by the new design in #104.

**Comment by maxcountryman** [comment]
This is addressed in `v0.7.0`.
